schema/articles.js

```diff
@@ -97,5 +97,16 @@ export default {
         updatedAt: { type: 'date' },
       },
     },
+
+    articleType: { type: 'keyword' }, // TEXT, IMAGE, VIDEO, AUDIO
+
+    // There will be an attachment describing the file when articleType is not TEXT
+    attachment: {
+      type: 'nested',
```

Comment:
Question: will there be multiple attachments? If not, we can consider using `attachmentUrl` + `attachmentHash` instead.

Reply:
There won't be multiple attachments. I'll use `attachmentUrl` and `attachmentHash`.